### PR TITLE
DHFPROD-9811: Additional Error Validation For Create New List

### PR DESCRIPTION
**Pull request by amejiaq**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [n/a] Added to Release Wiki

### CODE REVIEW

**Review by MSBrignoni — APPROVED** [review]

**Review by briantang — CHANGES_REQUESTED** [review]
Looks good @amejiaq. One thing with the existing duplicate value error message:

Screenshot 2023-01-30 at 10 06 08 AM

Can we upgrade the text to as follows?

The value **"duplicate_example"** already exists in this list. Please enter a unique value to ignore.

**Review by briantang — CHANGES_REQUESTED** [review]
Screenshot 2023-01-30 at 11 05 42 PM

Can we add a period to the end of the error message?

Also, sorry for the confusion, but actually I think the referenced value should only be bolded with no surrounding quotes, to be consistent with other parts of the app (duplicate entity name, property name, etc.).

For example,

The value **a** already exists in this list.

**Comment by amejiaq** [comment]
@briantang  Fixed 🛠

File: marklogic-data-hub-central/ui/src/components/entities/matching/list-modal/list-modal.test.tsx [hunk 1]
```diff
@@ -48,13 +50,62 @@ describe("Test input validation values to ignore", () => {
...
+      confirmAction={() => {}}
+      checkIfExistInList={() => { return false; }}
+      updateListValues={() => {}}
+      listValues={["One"]}
+    />);
+    expect(screen.queryAllByText(errorText)).toHaveLength(0);
+    const inputListValues =screen.getByPlaceholderText("Enter values to remove");
+    fireEvent.change(inputListValues, {target: {value: "ab cd"}});
+    fireEvent.click(screen.getByRole("option"));
+    expect(screen.queryAllByText(errorText)).toHaveLength(1);
```
Comment (briantang) [inline marklogic-data-hub-central/ui/src/components/entities/matching/list-modal/list-modal.test.tsx:108]: If possible, can we also add a check here to verify the latest duplicate values error?

**Review by briantang — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/src/components/entities/matching/list-modal/list-modal.test.tsx [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import {fireEvent} from "@testing-library/dom";
 import {cleanup, render, screen} from "@testing-library/react";
+import {getSubElements} from "@util/test-utils";
```
Comment (briantang) [inline marklogic-data-hub-central/ui/src/components/entities/matching/list-modal/list-modal.test.tsx:3]: Nice :)

**Review by briantang — APPROVED** [review]

**Review by littlecherro — APPROVED** [review]

**Pull request merged**